examples/sets/subset.pvl

```diff
@@ -0,0 +1,18 @@
+// -*- tab-width:2 ; indent-tabs-mode:nil -*-
```

Comment:
Can we add asserts for the not contains cases? such as asssert !({a,b,c} < {a,b,d,e,f})

Reply:
Yes, I will add one for sets and one for bags (in the next commit).